src/Microsoft.AspNetCore.Authentication.OpenIdConnect/Events/AuthorizationCodeReceivedContext.cs

```diff
@@ -17,29 +18,72 @@ public class AuthorizationCodeReceivedContext : BaseOpenIdConnectContext
         /// <summary>
         /// Creates a <see cref="AuthorizationCodeReceivedContext"/>
         /// </summary>
-        public AuthorizationCodeReceivedContext(HttpContext context, OpenIdConnectOptions options, AuthenticationProperties properties)
+        public AuthorizationCodeReceivedContext(HttpContext context, OpenIdConnectOptions options)
             : base(context, options)
         {
-            Properties = properties;
         }
 
         public AuthenticationProperties Properties { get; set; }
 
         /// <summary>
-        /// Gets or sets the 'code'.
+        /// Gets or sets the <see cref="JwtSecurityToken"/> that was received in the authentication response, if any.
         /// </summary>
-        public string Code { get; set; }
+        public JwtSecurityToken JwtSecurityToken { get; set; }
```

Comment:
Hmm, `IdToken` is used in the code to refer to the raw text value, e.g. OpenIdConnectMessage.IdToken.  `IdentityToken` isn't used anywhere so I don't know if it would help clarify. `SecurityToken` is used in a few places.

I'm going to leave it for now. There's not a clearly better name and it is still then name carried forward from Katana. You only noticed here because I moved it around in the file.